Update for zephyr

mjs513 opened the pull request:
Updated so GFX works with either Zephyr or mbed.

Requires the update to Arduino_GigaDisplay library to work with zephyr.  @KurtE and I have tested this and used it.  PS - for touch to work you will need to include the @KurtE's PR: https://github.com/arduino/ArduinoCore-zephyr/pull/134.

So please incorporate that library as well.

mjs513 committed:
Update for zephyr

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/arduino-libraries/Arduino_GigaDisplay_GFX?pullRequest=10) <br/>All committers have signed the CLA.

mjs513 committed:
Added 2 new functions

Added drawGrayscaleBitmapScaled and drawRGBBitmapScaled functions to allow scaling of 320x240 images such on the giga display.  Simplifies code in sketches

leonardocavagnis commented:
Hi @mjs513, thank you for your work on this!

I have moved this implementation to #11, where I’ve updated the code to integrate with the new [Arduino_Video](https://github.com/arduino-libraries/Arduino_Video) library (WIP & private).

Closing this in favor of #11.
